PSReadLine/Accessibility.cs

```diff
@@ -10,14 +11,73 @@ internal class Accessibility
     {
         internal static bool IsScreenReaderActive()
         {
-            bool returnValue = false;
-
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                return IsAnyWindowsScreenReaderEnabled();           // Check for macOS VoiceOver
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
+                return IsVoiceOverEnabled();
+
+            // TODO: Support Linux per https://code.visualstudio.com/docs/configure/accessibility/accessibility
+            return false;
+        }
+
+        private static bool IsAnyWindowsScreenReaderEnabled() {
+            // The official way to check for a screen reader on Windows is
+            // SystemParametersInfo(SPI_GETSCREENREADER, ...) but it doesn't
+            // detect the in-box Windows Narrator and is otherwise known to be
+            // problematic.
+            //
+            // Unfortunately, the alternative method used by Electron and
+            // Chromium, where the relevant screen reader libraries (modules)
+            // are checked for does not work in the context of PowerShell
+            // because it relies on those applications injecting themselves into
+            // the app. Which they do not because it's not a windowed app, so

```

Comment:
> Which they do not because it's not a windowed app

Do you mean "PowerShell" is not a windowed app here? It'd be great if you can reword the comments here a little to make it clearer.

Reply:
Correct.

Reply:
Minor change based on our conversation.
```suggestion
            // the app. Which they do not because PowerShell not a windowed app, so
```